spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-bus-config-sample/spring-cloud-gcp-pubsub-bus-config-sample-test/src/test/java/com/example/LocalSampleAppIntegrationTest.java

```diff
@@ -143,7 +140,7 @@ public void closeResources() throws IOException {
 
 	}
 
-	private void startConfigServer() throws IOException {
```

Comment:
Going from `private` to package-private is relaxing visibility. Generally, it's best to have the least visibility that gets the job done. These private methods should stay private, unless there is a reason to change them.

Reply:
I had this in mind too but I thought uniformity across all the integration tests would be a better choice. No worries, I'll revert this change.